Want It

dbfjru opened the pull request:
## 변경사항

---
Want It

### 변경의 종류 (여러 가지 선택 가능)

- [ ] 새로운 기능
- [ ] 버그 수정
- [ ] 문서 업데이트
- [ ] 테스트 추가
- [ ] 기타(설명 추가)

## 체크리스트

- [ ] ERD설계와 API설계를 잘 적용하여 개발하였는가?
- [ ] 테스트는 모든 경우를 고려하였는가?
- [ ] 코딩 컨벤션을 준수하였는가?
- [ ] 예외 처리가 잘 되어 있는가?

dbfjru committed:
Merge pull request #25 from DCNsProjects/chore

Chore

HEEEUN9812 committed:
:sparkles: feat : 입찰 조회 기능 추가

HEEEUN9812 committed:
Merge pull request #26 from DCNsProjects/feat/bid

:sparkles: feat : 입찰 조회 기능 추가

HEEEUN9812 committed:
:sparkles: feat : 포인트 내역 페이징 기능 추가

YOOSUNAH committed:
✨ feat : 이메일 인증 기능 구현

YOOSUNAH committed:
Merge pull request #27 from DCNsProjects/email

✨ feat : 이메일 인증 기능 구현

YOOSUNAH committed:
♻️ refactor : 필요없는 주석 삭제

HEEEUN9812 committed:
:adhesive_bandage: fix : 페이지 1번부터 출력하도록 수정

HEEEUN9812 committed:
:wrench: chore : 충돌 해결

dbfjru committed:
🔧 chore : scheduler, event 패키지 위치이동

dbfjru committed:
Merge remote-tracking branch 'origin/dev' into feat/aop

HEEEUN9812 committed:
:sparkles: feat : 포인트 내역 조건 검색 추가

YOOSUNAH committed:
♻️ refactor : 설명 추가, 리팩토링

HEEEUN9812 committed:
:adhesive_bandage: fix : pointLog status 인덱스 추가

YOOSUNAH committed:
Merge pull request #29 from DCNsProjects/feat/user

Feat/user

dbfjru committed:
✨ feat : 금지품목 키워드 검열 AOP 구현

itemDescription을 기준으로 금지 키워드를 검열
영어로 써있는 경우 소문자로 변환하여 검열

dbfjru committed:
Merge remote-tracking branch 'origin/dev' into feat/aop

HEEEUN9812 committed:
:adhesive_bandage: fix : 페이지 사이즈, 현재 페이지, 총 페이지 반환값 추가

HEEEUN9812 committed:
Merge pull request #28 from DCNsProjects/feat/point

Feat/point

YOOSUNAH committed:
🩹 fix : 카카오 로그인 기능 과 이메일 인증기능이 build.gradle에서 충돌이 나서, 이를 해결함.
+ 회원 탈퇴 delete 메서드에서 patch메서드로 변경

dbfjru committed:
Merge pull request #30 from DCNsProjects/feat/aop

Feat/aop

YOOSUNAH committed:
✨ feat : 리프레쉬 토큰을 레디스에 저장하는 방식으로 로그인 부분 기능 구현 수정

YOOSUNAH committed:
♻️ refactor :  리프레쉬 토큰을 레디스에 저장하는 방식으로 로그인 부분 기능 구현 리팩토링 및 설명 추가

dbfjru committed:
Merge pull request #31 from DCNsProjects/users

🩹 fix : 카카오 로그인 기능 과 이메일 인증기능이 build.gradle에서 충돌이 나서, 이를 해결함.

dbfjru committed:
Merge pull request #32 from DCNsProjects/user

User

YOOSUNAH committed:
✨ feat : 구글이메일로 인증하기 기능 구현(네이버에서 구글이메일로 변경)

dbfjru committed:
✨ feat : 실시간 입찰가격 송출 서비스 구현

dbfjru committed:
✨ feat : 경매 오픈 로직 구현

YOOSUNAH committed:
Merge pull request #33 from DCNsProjects/user/email

✨ feat : 구글이메일로 인증하기 기능 구현(네이버에서 구글이메일로 변경)

dbfjru committed:
Merge pull request #35 from DCNsProjects/feat/scheduler

dbfjru committed:
Merge pull request #34 from DCNsProjects/feat/liveBid

dbfjru committed:
✨ feat : 경매 낙찰 메일 발송 구현

feel00k0810 committed:
:see_no_evil: update : update .gitignore

feel00k0810 committed:
:heavy_plus_sign: update : Add Firebase Dependency

feel00k0810 committed:
:sparkles: Add : Add FCM initializer

feel00k0810 committed:
:construction: 작업중

feel00k0810 committed:
✨ feat : Add Successful bid notification

feel00k0810 committed:
✨ update : Update Successful bid notification Scheduler

feel00k0810 committed:
➖ Remove : Remove a dependency

feel00k0810 committed:
🔧 update : Changing the Firebase SDK Version

feel00k0810 committed:
🔧 chore : Schedule Time change

HEEEUN9812 committed:
:adhesive_bandage: fix : uri 수정

dbfjru committed:
Merge pull request #37 from DCNsProjects/feat/emailToWinner

✨ feat : 경매 낙찰 메일 발송 구현

feel00k0810 committed:
Merge pull request #39 from DCNsProjects/feat/point

:adhesive_bandage: fix : uri 수정

dbfjru committed:
Merge pull request #38 from DCNsProjects/feat/FCM

Feat/fcm

dbfjru committed:
🩹 fix : Removing deprecated method

feel00k0810 committed:
Merge pull request #41 from DCNsProjects/feat/FCM

🩹 fix : Removing deprecated method

snt2525 commented on build.gradle:
```diff
@@ -62,10 +62,26 @@ dependencies {
 
+    // Redis
+    implementation 'org.springframework.boot:spring-boot-starter-data-redis'
+    implementation 'org.redisson:redisson-spring-boot-starter:3.27.1'
```
[필수]
redis 사용 안하시면 관련 의존성은 제거해주세요~

YOOSUNAH replied:
redis 사용합니다! (refreshtoken 관리, 이메일 인증 코드 관리)

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/auctionItem/controller/MyAuctionItemController.java:
```diff
@@ -1,10 +1,98 @@
+        @AuthenticationPrincipal UserDetailsImpl userDetails
+    ) {
+        myAuctionItemService.createAuctionItem(request, userDetails.getUser());
+        return ResponseDto.of(HttpStatus.CREATED, null);
```
ResponseEntity<ResponseDto<Void>>로 응답 타입을 내려주시는 것 보다
ResponseEntity<ResponseDto<Long>>으로 옥션 아이템을 생성했을 때 DB에서 응답해준 id를 response해주는 게 더 좋을 것 같습니다.
`
        return ResponseDto.of(HttpStatus.CREATED, id);
`

YOOSUNAH committed:
:bug: acceessToken에서 user정보(userId)를 가지고 오지 못하는 문제를 해결 + email 인증 코드 만료시간 단축

YOOSUNAH committed:
:recycle: repository 통일

YOOSUNAH committed:
:recycle: topic 이름 추가

dbfjru committed:
Merge pull request #42 from DCNsProjects/user

:bug: acceessToken에서 user정보(userId)를 가지고 오지 못한 문제 해결

dbfjru committed:
✨ feat : 전체 경매 목록 조회 페이징 구현

YOOSUNAH committed:
:bug: admin으로 회원가입 되도록 수정

YOOSUNAH committed:
:sparkles: admin만 접근 가능한 api 생성 (user 전체 조회)

YOOSUNAH committed:
:recycle: responseDTO에 담기

dbfjru committed:
✨ feat : 종료된 경매 목록 조회 페이징 구현

YOOSUNAH committed:
:recycle: email 타입리프 적용부분 수정

dbfjru committed:
Merge branch 'dev' into feat/auctionItem

YOOSUNAH committed:
:recycle: admin 이 user 전체 조회 부분 페이징 처리

YOOSUNAH committed:
:recycle: admin 이 user 전체 조회 부분 페이징 처리 - 전체 페이지 중 현재 페이지 표시 추가

YOOSUNAH committed:
Merge pull request #43 from DCNsProjects/admin

Admin  API 추가

dbfjru committed:
Merge pull request #44 from DCNsProjects/feat/auctionItem

Feat/auction item

YOOSUNAH committed:
:sparkles: 회원가입 테스트 코드 구현

YOOSUNAH committed:
:sparkles: 회원가입 테스트 코드 구현

YOOSUNAH committed:
:sparkles: 회원가입 실패 테스트 코드 구현

YOOSUNAH committed:
:sparkles: 로그인 테스트 코드 구현

YOOSUNAH committed:
:sparkles: 로그인 실패 테스트 코드 구현

YOOSUNAH committed:
:sparkles: 로그아웃 테스트 코드 구현

YOOSUNAH committed:
:sparkles: 회원 탈퇴 테스트 코드 구현

feel00k0810 committed:
:white_check_mark: 내 경매 아이템 생성 테스트 코드 추가

feel00k0810 committed:
Merge remote-tracking branch 'origin/dev' into test/myAuctionItem

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/admin/dto/UsersResponseDto.java:
```diff
@@ -0,0 +1,32 @@
+    private String address;
+    private Long kakaoId;
+
+    public UsersResponseDto(User user) {
```
[옵션]
생성자의 매개변수로 User 객체를 사용하는 것은 외부에서 User 객체를 전달해야 하므로 결합도가 높습니다. 생성자 대신에 정적 팩토리 메서드를 적용하여서 객체를 return 해주시는 것도 좋을 것 같습니다~!

`
public static UsersResponseDto fromUser(User user) {
        UsersResponseDto dto = new UsersResponseDto();
        dto.setUserId(user.getUserId());
        dto.setEmail(user.getEmail());
        dto.setPassword(user.getPassword());
        dto.setUsername(user.getUsername());
        dto.setNickname(user.getNickname());
        dto.setPhoneNumber(user.getPhoneNumber());
        dto.setAddress(user.getAddress());
        dto.setKakaoId(user.getKakaoId());
        return dto;
    }`

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/admin/dto/UserPageableResponseDto.java:
```diff
@@ -0,0 +1,18 @@
+
+@Getter
+@AllArgsConstructor
+public class UserPageableResponseDto {
```
[참고]
자바 17 이상을 쓰고 계신다면 record도 적용해보는 것도 좋을 것 같습니다.
https://www.dotnetnote.com/docs/java/records/

YOOSUNAH replied:
UserPageableResponseDto 에 적용했습니다! UserResponseDto 도 적용해보았습니다!

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/auctionItem/repository/AuctionItemQueryRepository.java:
```diff
@@ -0,0 +1,199 @@
+
+    private final JPAQueryFactory jpaQueryFactory;
+
+    public List<FinishedItemResponseDto> findWinningAuctionItems(Long userId) {
```
[필수]
Long 타입을 사용하신다는 의미는 Null 값이 넘어 올 수 있다는 의미로 null에 대한 예외처리를 하시거나, null이 넘어 올 가능성이 없다면 long인 프리미티브 타입으로 선언해주시는 것이 좋을 것 같습니다.

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/auctionItem/repository/AuctionItemQueryRepository.java:
```diff
@@ -0,0 +1,199 @@
+            .fetch();
+    }
+
+    public List<AuctionItem> findAllEndAuctionItems() {
```
[궁금] 
List<AuctionItem> findByStatusAndEndDateBefore(AuctionItemEnum status, LocalDateTime endDate); 
와같이 jpa 쿼리를 사용안하시는 이유가 있으실까요?????
AuctionItemQueryRepository 에 extends JpaRepository와 커스텀한 쿼리를 선언한 클래스를 같이 extend하여 구분해서 구현하면 더 깔끔할 것 같아서요..!!

참고: https://keencho.github.io/posts/spring-data-jpa-custom-querydsl-repository/

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/user/service/KakaoService.java:
```diff
@@ -0,0 +1,159 @@
+
+
+    public String kakaoLogin(String code) throws JsonProcessingException {
+        // 1. "인가 코드"로 "액세스 토큰" 요청
```
해당 클래스에 주석이 많은데요~ 불필요한 주석은 조금 제거작업 해주신는 것도 좋을 것 같습니다!
아래 글을 참고해주세요! 

참고: https://velog.io/@youngerjesus/%ED%81%B4%EB%A6%B0-%EC%BD%94%EB%93%9C-Chapter-4.-%EC%A3%BC%EC%84%9D

YOOSUNAH replied:
넵!

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/user/service/KakaoService.java:
```diff
@@ -0,0 +1,159 @@
+        body.add("redirect_uri", "http://localhost:8080/v1/users/kakao/callback");   // redirect URL
+        body.add("code", code);
+
+        RequestEntity<MultiValueMap<String, String>> requestEntity = RequestEntity
```
[필수]
요청 URL 생성이 이미 하드 코딩으로 되어있기 때문에 String kakaoTokenUrl = "https://kauth.kakao.com/oauth/token"; 로 선언해주셔도 괜찮을 것 같습니다. 그런데 url 그리고 아래의 grant_type,  client_id, redirect_uri와 같은 정보들은 프로퍼티 파일에 넣어서 관리해주시는 것이 좋을 것 같습니다.

HttpEntity<MultiValueMap<String, String>> requestEntity = new HttpEntity<>(body, headers);
ResponseEntity<String> response = restTemplate.exchange(kakaoTokenUrl, HttpMethod.POST, requestEntity, String.class);

YOOSUNAH replied:
넵넵!

snt2525 reviewed (commented):
틈틈히 출근 시에 시간되면 코드 봐볼게요. 세세히 보지는 못하지만... 우선 3분에 1 정도 얼추 본 것 같습니다. 1차적으로 본 곳 까지 먼저 코멘트 드리고 나머지도 보고 2 또는 3차로 리뷰 드리겠습니다.
고생하셨습니다~🚀🚀🚀👍🏻👍🏻👍🏻👍🏻👍🏻👍🏻

feel00k0810 committed:
:heavy_plus_sign: Add : Add a dependency

feel00k0810 committed:
:white_check_mark: Test : Add Test code

feel00k0810 committed:
:wrench: chore : Fix import

feel00k0810 committed:
:white_check_mark: Test : Add a single lookup test code

feel00k0810 committed:
:white_check_mark: Test : Add auction modification test code

feel00k0810 committed:
:white_check_mark: Test : Add auction deletion test code

feel00k0810 committed:
:white_check_mark: Test : Add Repository Test

HEEEUN9812 committed:
:white_check_mark: test : PointService 테스트 코드 추가

dbfjru committed:
🩹 fix : 경매 유찰된 경우 추가

YOOSUNAH committed:
:recycle: 코드 리뷰 반영

dbfjru committed:
Merge remote-tracking branch 'origin/dev' into feat/scheduler

HEEEUN9812 committed:
:white_check_mark: test : PointLogService 테스트 코드 추가

YOOSUNAH committed:
:sparkles:  user 업데이트 테스트 코드 생성 및 관련 코드 리팩토링

YOOSUNAH committed:
:sparkles:  user 비밀번호 수정 테스트 코드 생성

YOOSUNAH committed:
:sparkles:  user 비밀번호 수정 실패케이스 테스트 코드 생성

feel00k0810 committed:
:heavy_minus_sign: Remove : Remove a dependency

dbfjru committed:
Merge pull request #45 from DCNsProjects/test/myAuctionItem

MyAuctionItem 관련 테스트 코드 추가

HEEEUN9812 committed:
Merge pull request #46 from DCNsProjects/test/point

:white_check_mark: test : PointService 테스트 코드 추가

dbfjru committed:
✅ test : AuctionItemService 테스트 코드 추가

dbfjru committed:
Merge pull request #47 from DCNsProjects/feat/scheduler

Feat/scheduler

dbfjru committed:
Merge pull request #48 from DCNsProjects/test/pointLog

:white_check_mark: test : PointLogService 테스트 코드 추가

YOOSUNAH committed:
:wrench:  코드 리뷰 반영

YOOSUNAH committed:
:wrench:  코드 리뷰 반영

HEEEUN9812 committed:
:white_check_mark: test : BidService 테스트 코드 추가

YOOSUNAH committed:
Merge pull request #50 from DCNsProjects/test/users

Test/users

dbfjru committed:
Merge pull request #51 from DCNsProjects/feat/auctionItem

✅ test : AuctionItemService 테스트 코드 추가

YOOSUNAH committed:
:recycle: 테스트 코드 @Nested 로 관련 테스트 끼리 묶기

dbfjru committed:
✅ test : Scheduler 테스트 코드 추가

dbfjru committed:
🔧 chore : given 메소드 any~~ 로 인자 전달

YOOSUNAH committed:
:sparkles: admin API 회원 전체 조회 성공 테스트 구현 (+ Pageable 전달값 타입 문제로, 코드 리뷰 반영한 UserResponseDto record 타입에서 다시 class 타입으로 변경)

YOOSUNAH committed:
:wrench: 불필요한 주석 삭제

YOOSUNAH committed:
Update UserCommonTest.java

사용되지 않는 변수 삭제

YOOSUNAH committed:
Update AdminServiceTest.java

YOOSUNAH committed:
:recycle: ResponseDto 코드 리뷰 반영해서 record 타입으로 변경 , 관련 부분들 함께 수정

YOOSUNAH committed:
Merge remote-tracking branch 'origin/test/user' into test/user

YOOSUNAH committed:
:bug: 코드 리뷰 반영하며, 카카오 서비스 오류 수정

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/auctionItem/service/AuctionItemService.java:
```diff
@@ -1,10 +1,66 @@
+
+    public AuctionItemResponseDto getAuctionItem(Long auctionItemId) {
+        AuctionItem auctionItem = auctionItemRepository.findById(auctionItemId).orElseThrow(
+            () -> new IllegalArgumentException("존재하지 않는 경매상품 입니다.")
```
IllegalArgumentException 대신 예외코드를 직접 정의하시는걸 추천드립니다!
아래의 클린코드 책에서 나왔던 관련 블로그랑 커스텀 예외처리하는 방법에 대한 문서 전달드립니다~

클린코드 관련 :https://amazingguni.github.io/blog/2016/05/Clean-Code-7-%EC%98%A4%EB%A5%98-%EC%B2%98%EB%A6%AC

커스텀 예외처리: https://velog.io/@chiyongs/Exception%EC%9D%84-Custom%ED%95%98%EA%B2%8C-%EB%A7%8C%EB%93%A4%EC%96%B4%EC%84%9C-%EC%82%AC%EC%9A%A9%ED%95%B4%EB%B3%B4%EC%9E%90

snt2525 commented on src/main/java/io/dcns/wantitauction/domain/auctionItem/service/MyAuctionItemService.java:
```diff
@@ -0,0 +1,78 @@
+        return auctionItemQueryRepository.findAllMyAuctionItems(userId);
+    }
+
+    public MyAuctionItemsResponseDto getAuctionItem(Long auctionItemId, Long userId) {
```
레퍼런스 타입의 변수 많이 사용하셨는데요. 특별한 이유가 있는게 아니시라면 프리미티브 타입의 변수를 사용하시는 것을 추천드립니다.
레퍼런스 타입의 변수를 사용하셔야한다면 항상 null에 대한 예외처리도 함께 생각해주셔야합니다.

관련문서: https://n1tjrgns.tistory.com/290

snt2525 commented on src/main/java/io/dcns/wantitauction/global/jwt/RefreshTokenRepository.java:
```diff
@@ -0,0 +1,38 @@
+
+    private final RedisTemplate<String, String> redisTemplate;
+    @Resource(name = "redisTemplate")
+    private ValueOperations<String, String> valueOperations;
```
의존성 주입 시에 RedisTemplate 및 ValueOperations를 필드로 직접 주입하고 계신데요. 
@RequiredArgsConstructor과 충돌이 일어날 수 있기 때문에 @Qualifier("redisTemplate") 를 활용한 생성자 주입 방식을 사용하시는 것도 좋을 것 같습니다.

`
public RefreshTokenRepository(RedisTemplate<String, String> redisTemplate, @Qualifier("redisTemplate") ValueOperations<String, String> valueOperations) {
        this.redisTemplate = redisTemplate;
        this.valueOperations = valueOperations;
    }
`

YOOSUNAH replied:
넵!

YOOSUNAH committed:
:bug: 카카오 회원가입 실패 오류 수정 (카카오로 회원가입시, nickname을 받지 않을 경우를 추가)

snt2525 reviewed (commented):
오늘은 시간 여유가 많지 않아서 많이 못봤습니다ㅠㅠ 
다음 시간에 여유 되면 틈틈히 계속 봐보겠습니다~
이번주도 화이팅하세요!!

dbfjru committed:
Merge pull request #53 from DCNsProjects/feat/scheduler

✅ test : Scheduler 테스트 코드 추가

dbfjru committed:
Merge pull request #55 from DCNsProjects/test/user

✅ test : adminService 테스트 코드 추가

dbfjru committed:
♻️ refactor : 내 경매 리스트 조회 페이징 구현

dbfjru committed:
✅ test : 내 경매 리스트 페이징 테스트 추가

dbfjru committed:
♻️ refactor : 내 낙찰 상품 목록 조회 페이징 구현

dbfjru committed:
✅ test : 내 낙찰 경매 목록 조회 테스트 추가

dbfjru committed:
Merge remote-tracking branch 'origin/dev' into feat/auctionItem

dbfjru committed:
Merge branch 'dev' into test/bid

dbfjru committed:
Merge pull request #52 from DCNsProjects/test/bid

:white_check_mark: test : BidService 테스트 코드 추가

dbfjru committed:
Merge remote-tracking branch 'origin/dev' into feat/auctionItem

dbfjru committed:
🐛 bug : lombok 중복 수정

HEEEUN9812 committed:
Merge pull request #56 from DCNsProjects/feat/auctionItem

Feat/auction item

dbfjru committed:
🐛 bug : Json Parse Error 해결

dbfjru committed:
🔧 chore : scheduler Off

dbfjru committed:
🔧 chore : API 오타 수정

YOOSUNAH committed:
♻️ 코드 리뷰 반영, @Qualifer로 빈을 명시적으로 등록

dbfjru committed:
Merge remote-tracking branch 'origin/dev' into feat/liveBid

YOOSUNAH committed:
Merge pull request #58 from DCNsProjects/feat/user

♻️ 코드 리뷰 반영, @Qualifer로 빈을 명시적으로 등록

dbfjru committed:
🐛 bug : SSE 실시간 호가 서비스 로직 변경

기존의 로직은 For문 안에 try~catch 가 존재해서, 반복문 처리 도중 예외가 발생하게 되면, 뒤 순번들에게는 실시간 서비스가 발송되지 않는 문제 발생했음.
이를 해결하기 위해 반복문을 제거하고 stream 으로 처리

dbfjru committed:
Merge remote-tracking branch 'origin/dev' into feat/liveBid

dbfjru committed:
Merge pull request #59 from DCNsProjects/feat/liveBid

사소한 오류 수정

YOOSUNAH committed:
:sparkles: 경매상품 조회시 , 레디스를 이용한 캐싱 처리

HEEEUN9812 committed:
:sparkles: feat : BidService createBid 분산락 적용 추가

dbfjru committed:
♻️ refactor : SSE 로직 간소화

기존 코드에서 중복되는 코드 통합

YOOSUNAH committed:
:recycle: 리팩토링

dbfjru committed:
🐛 bug : SseEmitter 존재하지 않을 때 수정

dbfjru committed:
Merge pull request #61 from DCNsProjects/feat/bid

:sparkles: feat : BidService createBid 분산락 적용 추가

dbfjru committed:
Merge pull request #62 from DCNsProjects/feat/liveBid

♻️ refactor : SSE 로직 간소화

dbfjru committed:
✨ feat : 예정된 경매 목록 조회 페이징 구현

dbfjru committed:
✅ test : 예정된 경매 목록 조회 테스트 추가

feel00k0810 committed:
:zap: Feat : CI/CD 개발중

YOOSUNAH committed:
:recycle: 프론트 통신테스트 - 토큰 body에도 담아주기

dbfjru committed:
🐛 bug : 내 경매 목록 조회 오류 해결

dbfjru committed:
✨ feat : 진행중인 경매 목록 페이지 조회

dbfjru committed:
🔧 chore : 어노테이션 위치 변경

dbfjru committed:
✅ test : 진행중인 경매목록 조회 테스트

feel00k0810 committed:
:art: Update : CI/CD yml 파일 구조 개선

dbfjru committed:
Merge pull request #63 from DCNsProjects/feat/auctionItem-nhs

Feat/auction item nhs

feel00k0810 committed:
Merge remote-tracking branch 'origin/dev' into feat/CICD

YOOSUNAH committed:
:recycle: 백엔드 프론트 통신테스트 - email 인증 요청 타입 변경, security 허용

YOOSUNAH committed:
Merge pull request #60 from DCNsProjects/feat/auctionItem

:sparkles: 경매상품 조회시 , 레디스를 이용한 캐싱 처리

feel00k0810 committed:
Merge pull request #64 from DCNsProjects/feat/CICD

CI/CD 설정 반영

feel00k0810 committed:
:wrench: chore : 코드 변경

feel00k0810 committed:
Merge pull request #65 from DCNsProjects/feat/CICD

:wrench: chore : 코드 변경

feel00k0810 committed:
:wrench: chore : 코드 변경

feel00k0810 committed:
Merge pull request #66 from DCNsProjects/feat/CICD

yml 파일 코드 변경

feel00k0810 committed:
:wrench: chore : 코드 변경

feel00k0810 committed:
Merge pull request #67 from DCNsProjects/feat/CICD

:wrench: chore : 코드 변경

feel00k0810 committed:
:wrench: chore : 코드 변경

feel00k0810 committed:
Merge pull request #68 from DCNsProjects/feat/CICD

:wrench: chore : 코드 변경

feel00k0810 committed:
:wrench: chore : FCM

feel00k0810 committed:
Merge remote-tracking branch 'origin/dev' into feat/CICD

YOOSUNAH committed:
:sparkles:  백엔드 프론트 통신테스트 - user 정보 업데이트시, 현재 정보 보여주기 위해 api 추가

YOOSUNAH committed:
:recycle:  백엔드 프론트 통신테스트 - 변수명 변경

HEEEUN9812 committed:
:sparkles: feat : WebCorsConfig 추가

HEEEUN9812 committed:
:wrench: chore : PointResponseDto에 nickName 추가

HEEEUN9812 committed:
Merge branch 'dev' into feat/bid

dbfjru committed:
Merge pull request #70 from DCNsProjects/feat/bid

Feat/bid

HEEEUN9812 committed:
:wrench: chore : PointResponseDto에 username 추가 및 처음 포인트 조회시 포인트 객체 생성 로직 추가

HEEEUN9812 committed:
Revert ":wrench: chore : PointResponseDto에 username 추가 및 처음 포인트 조회시 포인트 객체 생성 로직 추가"

This reverts commit 6fefca4f62cf7857f032236d365d0187125a6af5.

HEEEUN9812 committed:
Merge branch 'dev' into feat/point

HEEEUN9812 committed:
:wrench: chore : PointResponseDto에 username 추가 및 처음 포인트 조회시 포인트 객체 생성 로직 추가

feel00k0810 committed:
Merge remote-tracking branch 'origin/dev' into feat/CICD

feel00k0810 committed:
:sparkles: Feat : 환경 변수 수정

feel00k0810 committed:
:heavy_plus_sign: Add : 스프링 클라우드 스타터 디펜던시 추가

feel00k0810 committed:
:wrench: chore : 환경변수 추가

feel00k0810 committed:
:sparkles: Feat : S3 관련 기능 추가

- 사용처 추가 필요

feel00k0810 committed:
:wrench: chore : fileURL 수정

feel00k0810 committed:
:wrench: chore : CI/CD 브랜치 변경

YOOSUNAH committed:
:recycle:  백엔드 프론트 통신테스트 - 로그인시 header 에 응답 추가 삭제

feel00k0810 committed:
Merge pull request #69 from DCNsProjects/feat/CICD

CI/CD FCM 환경변수 추가

feel00k0810 committed:
Merge pull request #72 from DCNsProjects/feat/S3

S3 관련 기능 추가

dbfjru committed:
Merge pull request #71 from DCNsProjects/feat/point

Feat/point

YOOSUNAH committed:
Merge remote-tracking branch 'origin/dev' into feat/auctionItem

# Conflicts:
#	src/main/java/io/dcns/wantitauction/domain/user/service/UserService.java

dbfjru committed:
Merge pull request #77 from DCNsProjects/feat/auctionItem

Feat/auction item

HEEEUN9812 committed:
:sparkles: feat : 입찰 수 상위 3개 조회 기능 추가

HEEEUN9812 committed:
:sparkles: feat : 카테고리 컬럼 추가 및 진행중인 경매에서 카테고리 필터 기능 추가

HEEEUN9812 committed:
:wrench: chore : 메소드명 변경 및 dto 수정

HEEEUN9812 committed:
:wrench: chore : api 수정

HEEEUN9812 committed:
Merge pull request #78 from DCNsProjects/feat/bid

:sparkles: feat : 입찰 수 상위 3개 조회 기능 추가

HEEEUN9812 committed:
Merge branch 'dev' into feat/auctionItem

HEEEUN9812 committed:
:wrench: chore : TopAcutionItemsResponseDto에 category 추가

YOOSUNAH committed:
:recycle:  백엔드 프론트 통신테스트 - 내가 찜한(좋아요한) 상품 조회 API 수정

YOOSUNAH committed:
:recycle:  백엔드 프론트 통신테스트 - 내가 찜한(좋아요한) 상품 조회 API 수정 (추가로, 좋아요 취소한 상품 조회 api 추가 생성)

YOOSUNAH committed:
:recycle:  백엔드 프론트 통신테스트 - 내가 찜한(좋아요한) 상품 조회 API 수정 (추가로, 좋아요 취소한 상품 조회 api 추가 생성)

YOOSUNAH committed:
:recycle:  백엔드 프론트 통신테스트 - url endpoint s 추가

dbfjru committed:
Merge pull request #80 from DCNsProjects/feat/auctionItem

Feat/auctionItem 카테고리 컬럼 추가 및 진행중인 경매에서 카테고리 필터 기능 추가

YOOSUNAH committed:
:recycle:  백엔드 프론트 통신테스트 - 내가 입찰한 경매상품 목록 -응답 추가

YOOSUNAH committed:
:recycle: 로그 삭제

YOOSUNAH committed:
Merge pull request #81 from DCNsProjects/like

내가 찜한(좋아요한) 상품 조회 API 수정

dbfjru committed:
🔧 chore : @JsonFormat 불필요하여 제거

dbfjru committed:
Merge pull request #91 from DCNsProjects/feat/auctionItem-nhs

🔧 chore : @JsonFormat 불필요하여 제거

dbfjru committed:
🐛 bug : SSE 관련 구조 변경

dbfjru committed:
Merge pull request #92 from DCNsProjects/feat/liveBid

🐛 bug : SSE 관련 구조 변경

dbfjru committed:
✨ feat : 경매상품의 최고 호가 조회 API 추가

dbfjru committed:
Merge pull request #93 from DCNsProjects/feat/bid

✨ feat : 경매상품의 최고 호가 조회 API 추가

HEEEUN9812 committed:
:sparkles: feat : 입찰 조회 pageable 적용

HEEEUN9812 committed:
:wrench: chore : 코드 수정

HEEEUN9812 committed:
:wrench: chore : 코드 수정

HEEEUN9812 committed:
:wrench: chore : 코드 수정

dbfjru committed:
init : localhost 환경 설정

dbfjru committed:
Merge pull request #94 from DCNsProjects/feat/bid

:sparkles: feat : 입찰 조회 pageable 적용

dbfjru committed:
Merge pull request #95 from DCNsProjects/feat/point

:wrench: chore : 코드 수정

HEEEUN9812 committed:
:white_check_mark: test : BidServiceTest 수정

dbfjru committed:
✨ feat : imageURL 필드 추가

dbfjru committed:
🔧 chore : 미사용 기능 삭제

dbfjru committed:
🔧 chore : 디렉토리 정리

dbfjru committed:
✨ feat : AWS S3 관련 기능 추가

dbfjru committed:
✨ feat : File input 처리할 Util 생성

dbfjru committed:
✨ feat : 이미지 포함한 경매 상품 생성 기능

dbfjru committed:
🔧 chore : 줄바꿈 추가

dbfjru committed:
Delete application.yml

dbfjru committed:
Merge pull request #103 from DCNsProjects/debug/s3-nhs

Debug/s3 nhs

dbfjru committed:
:bug: 응답 header에 담긴 token 프론트에서 사용할 수 있게 수정

dbfjru committed:
🔧 chore : imageUrl 필드 추가

dbfjru committed:
🔧 chore : imageUrl 필드 추가

dbfjru committed:
🔧 chore : 미사용 메소드 삭제

dbfjru committed:
Merge pull request #105 from DCNsProjects/feat/auctionItem-nhs

Feat/auction item nhs

YOOSUNAH committed:
Merge remote-tracking branch 'origin/dev' into test

dbfjru committed:
Merge pull request #106 from DCNsProjects/test

 응답 header에 담긴 token 프론트에서 사용할 수 있게 수정

dbfjru committed:
✅ test : test 변경

HEEEUN9812 committed:
Merge pull request #107 from DCNsProjects/feat/auctionItem-nhs

✅ test : test 변경

HEEEUN9812 committed:
:white_check_mark: test : BidServiceTest 수정

dbfjru committed:
Merge pull request #108 from DCNsProjects/test/bid

Test/bid BidServiceTest 수정

HEEEUN9812 committed:
:white_check_mark: test : PointServiceTest 수정

dbfjru committed:
Merge pull request #109 from DCNsProjects/test/point

:white_check_mark: test : PointServiceTest 수정

HEEEUN9812 committed:
:adhesive_bandage: fix : BidResponseDto에 imageUrl 추가

HEEEUN9812 committed:
:wrench: chore : 간단 코드 정리

HEEEUN9812 committed:
Merge pull request #112 from DCNsProjects/feat/bid

Feat/bid  BidResponseDto에 imageUrl 추가

YOOSUNAH committed:
:recycle: image url 응답 추가 , 사용하지 않는 api 삭제

dbfjru committed:
Merge pull request #113 from DCNsProjects/like

:recycle: image url 응답 추가 , 사용하지 않는 api 삭제

dbfjru committed:
🩹 bug : 이미지 url 받아올 수 있게 변경

HEEEUN9812 committed:
Merge pull request #114 from DCNsProjects/feat/bid

🩹 bug : 이미지 url 받아올 수 있게 변경

HEEEUN9812 committed:
:tada: init : submodule

HEEEUN9812 committed:
Revert ":tada: init : submodule"

This reverts commit 5cc78e5fbd00443c0bc7ac5bdabdac1d792e24a8.

HEEEUN9812 committed:
Merge pull request #126 from DCNsProjects/feat/submodule

Feat/submodule

HEEEUN9812 committed:
:adhesive_bandage:  fix : build.gradle 수정

dbfjru committed:
Merge pull request #127 from DCNsProjects/feat/submodule

:adhesive_bandage:  fix : build.gradle 수정

HEEEUN9812 committed:
submodule 실패

HEEEUN9812 committed:
Merge pull request #130 from DCNsProjects/fix

submodule 실패

HEEEUN9812 committed:
submodule 삭제

dbfjru committed:
Merge pull request #131 from DCNsProjects/fix

submodule 삭제
